### PR TITLE
Edits to Itinerary

### PR DESCRIPTION
**Pull request by RoAnnC1**

### CODE REVIEW

File: src/api-reference/travel/itinerary-v4/v4.itinerary.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ The Itinerary API provides clients and authorized partners access to travel itin
 ## <a name="limitations"></a>Limitations
 
 * This API is only available to clients and partners who have been granted access by SAP Concur. Access to this documentation does not provide access to the API. This API is only available in US and EMEA data centers.
-* Events are only raised on a go-forward basis. If you need access to historical data, see the [retrieve trips endpoint](#retrieve-trips)
+* Events are only raised on a go-forward basis. If you need access to historical data, see the [retrieve trips endpoint](#retrieve-trips).
```
Comment (joelmbell) [inline src/api-reference/travel/itinerary-v4/v4.itinerary.md:12]: Just noticed I didn't update the link name for the element on line 2012 to #retrieve-trips. It's currently #get-trips so this link will not work as expected. Should I push up a fix for this?

Comment (RoAnnC1) [reply src/api-reference/travel/itinerary-v4/v4.itinerary.md:12]: I'll fix it since I have a pending PR.

**Review by joelmbell — DISMISSED** [review]

File: src/api-reference/travel/itinerary-v4/v4.itinerary.md (hunk 1 above)
Comment (RoAnnC1) [reply src/api-reference/travel/itinerary-v4/v4.itinerary.md:12]: Added, I'll go ahead and have these changes merged and anything else you come up with you can do a PR for.

**Review by jonikamelcher — APPROVED** [review]
:shipit:

**Pull request merged**